README.md

```diff
@@ -1,229 +1,60 @@
-- [ ] Configure default locale to `en_US.UTF-8`
-- [ ] Configure network scripts to use DHCP and enable them
-  Although not, strictly speaking, needed since kernel already has IP address and gateway this allows DHCP hooks to be called for setting hostname, etc
-- [ ] Install custom DHCP hook for hostname to set entry in `/etc/hosts` for software using `getent_r` to get hostname
-- [ ] Install scripts to fetch SSH keys
-- [ ] Install scripts to fetch kernel modules
-- [ ] Install scripts to connect and/or mount NBD volumes
-- [ ] Install scripts to manage NBD root volume
-- [ ] Disable all physical TTY initialization
-- [ ] Enable STTY @ 9600 bps
-
-Before making the image public, do not forget to check it boots, stops and restarts from the OS without any error (most notably kernel) since a failure could lead to deadlocked instances.
+### Docker-based images
 
+Using the tools contained in this repository, it is possible to create Scaleway images from Docker images. At its core, this process uses the tools we explored above and is pretty straightforward to use: 
+  1. Clone this repository and the repository of the image (if there is one) on a Scaleway instance
+  2. In your terminal, navigate to the image-tools directory, and type `make IMAGE_DIR=<the image's directory> scaleway_image`
+  3. ???
+  4. Profit !
 
-# How to download the common scripts on a target image
+As a first example, let's simply rebuild our Ubuntu Xenial image without changing anything. Note that this is not needed to build your own Docker-based images, and is shown here only for demonstration purposes.
 
-Those scripts are mainly used in the base image (distrib) but can sometimes be useful in the app images (inherited from distrib).o
-
-An example of usage in the [Ubuntu image](https://github.com/scaleway/image-ubuntu/blob/9cd0f287a1977a55b74b1a37ecb1c03c8ce55c85/14.04/Dockerfile#L12-L17)
-
-```bash
-wget -qO - https://j.mp/scw-skeleton | bash -e
-
-# Using upstart flavor
-wget -qO - https://j.mp/scw-skeleton | FLAVORS=upstart bash -e
-# Using sysvinit, docker-based and common flavors
-wget -qO - https://j.mp/scw-skeleton | FLAVORS=sysvinit,docker-based,common bash -e
-
-# Specific GIT branch
-wget -qO - https://j.mp/scw-skeleton | FLAVORS=upstart BRANCH=feature-xxx bash -e
-
-# Use curl
-curl -L -q https://j.mp/scw-skeleton | DL=curl bash -e
-
-# Alternative URL
-wget -qO - https://raw.githubusercontent.com/scaleway/image-tools/master/install.sh | ... bash -e
-```
+[![asciicast](https://asciinema.org/a/foiok7e9gWnyqKK6HbWQIaAQz.png)](https://asciinema.org/a/foiok7e9gWnyqKK6HbWQIaAQz)
 
-A running instance can be updated by calling the same commands.
-It is planned to to create packages (.deb) for distributions.
+Now that we've had a first taste, let's create a custom image based on Scaleway's Ubuntu image:
 
+[![asciicast](https://asciinema.org/a/qRVW81Ciqd1eeoki3Enyn3X7e.png)](https://asciinema.org/a/qRVW81Ciqd1eeoki3Enyn3X7e)
```

Comment:
Could it be possible for both screencasts to have a colorized command prompt? Otherwise, it's hard to identify where the prompt is and where the output of the command is.

Reply:
Sure ! I just never take the time to setup my prompts on servers.